tests/syft/lib/torch/nn/parameter/parameter_remote_method_api_suite_test.py

```diff
@@ -190,28 +196,33 @@ def test_all_allowlisted_parameter_methods_work_remotely_on_all_types(
     # Copy the UID over so that its easier to see they are supposed to be the same obj
     self_tensor_copy.id = self_tensor.id  # type: ignore
 
-    # TODO: This needs to be enforced to something more specific
-    args: Any
+    args: List[Any] = []
 
     # Step 4: Create the arguments we're going to pass the method on
     if _args == "None":
-        args = None
+        args = []
     elif _args == "self":
-        args = [th.nn.Parameter(self_tensor)]
+        args = [self_tensor]
     elif _args == "0":
-        args = 0
+        args = [PrimitiveFactory.generate_primitive(value=0)]
```

Comment:
Q: Shouldn't we also be able to pass Python values?

Reply:
We need to discuss the best way to handle this, either inject .serialize onto the primitives or detect them at the call site and wrap them where .serialize normally happens.

Until then, we need this to be able to test properties which send back other types than Tensor. As soon as we have the desired changes elsewhere we can just remove these from the test suite.